static/src/stylesheets/module/site-messages/_subscription-medium-banner.scss

```diff
@@ -388,3 +388,126 @@
 }
 
 /*********************** footer hacks END *********************/
+
+/*********************** gw Banner *********************/
```

Comment:
in an ideal world all this would be unnecessary:
a) we'd be able to find a consistent layout that accommodated different copy, packshots
b) tooling would control basic things like background colour

One for the future.